docs/howto/kubernetes-on-aks.md

```diff
@@ -0,0 +1,94 @@
+# Booting Kubernetes on Azure Kubernetes Service
+
+Azure Kubernetes Service (AKS) is an optimized container hosting solution that works with all the open source tools you
+know. Azure is great for Krustlet.
+
+If you don't yet have a Microsoft Azure account, start a trial with $200 of free credit
+[here](https://azure.microsoft.com/en-us/free/).
+
+## Prerequisites
+
+You should be able to run the `az` command, which is used to provision resources in the Azure cloud.
+
+Either [install Azure CLI](https://docs.microsoft.com/en-us/cli/azure/install-azure-cli?view=azure-cli-latest) to your
+computer or [open a Cloud Shell][cloudshell] by clicking this icon near
+the upper right of the Azure Portal: ![Azure Cloud Shell Icon](cloudshell.png)
```

Comment:
nit: could you add all images in a single directory?
I would assume that as we add more and more documentation, the root directory will get polluted.

Reply:
I support this nit

Reply:
I too support this nit